frontend/App.tsx

```diff
@@ -1,78 +1,134 @@
+  func1_resp: {},
+  func2_resp: {},
+  current: string,
+};
 
-  // useEffect(() => {
-  //   axios.get(func2_url)
-  //   .then(response => setFunc1Resp(response.data))
-  // },[]);
+export default class App extends Component<{}, State> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      phonenumber: "",
+      func1_resp: {},
+      func2_resp: {},
+      current: "",
+    }
+     // const url = "https://gner4se1je.execute-api.ap-south-1.amazonaws.com/Prod/func1"
+  }
 
-  function HomeScreen({ navigation }) {
-    return (
-      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-        <Text>Home Screen</Text>
-        <Button
-          title="SearchContacts"
-          onPress={() => navigation.navigate('SearchContacts')}
-        />
-        <Button
-          title="Chat with me"
-          onPress={() => navigation.navigate('ChatPanel')} />
-      </View>
-    );
+  componentDidMount(){
+    console.log("inside mount")
+    readData(STORAGE_KEY)
+      .then( (mphone) => {this.setState({phonenumber: mphone})})
+      .catch(() => console.log("inside catch"))
+      .finally(() => console.log("inside finally"));
+    
+    // axios.get(func1_url)
+    // .then(response => this.setState({func1_resp: response.data}))
   }
 
-  const Stack = createNativeStackNavigator();
+  HomeScreen({ navigation }) {
+    console.log("homescreen")
+      return (
+        <View>
+          <View style={styles.container}>
+            <Text style={[Regstyles.text,{fontSize: 30}]}>Home Screen</Text>
+            <View style={Regstyles.row}>
+              <View style={{padding: 10}}>
+                <Button
+                  title="SearchContacts"
+                  onPress={() => navigation.navigate('SearchContacts')}
+                />
+              </View>
+              <View style={{padding: 10}}>
+                <Button
+                  title="Chat with me"
+                  onPress={() => navigation.navigate('ChatPanel')} 
+                />
+              </View>
+            </View>
+          </View>
+          <View>
+            {/* <Pressable 
+              style={Regstyles.btn} 
+              onPress={() => removeData(STORAGE_KEY)}>
+              <MaterialIcons name="delete" size={25}/>
+            </Pressable> */}
+          </View> 
+          {/* Trigger to call clearstorage function (function is not working) 
+          Error: cannot read properties of undefined */}
+        </View>
+      );
+  }
 
-  return (
-    <NavigationContainer>
-      <Stack.Navigator>
-        <Stack.Screen name="Home" component={HomeScreen} />
-        <Stack.Screen name="SearchContacts" component={SearchContacts} />
-        <Stack.Screen name="ChatPanel" component={ChatPanel} />
-      </Stack.Navigator>
-    </NavigationContainer>
-  );
+  DefaultScreen({ navigation }){
+    return(
+      <View>
+        <View>
+          <Text style={[Regstyles.text,{fontSize: 30}]}>HiChat</Text>
```

Comment:
Font size constants should be added in `Constants.tsx` file so that we will have a single place to edit font sizes, this will help when we want to scale-up/scale-down elements

Reply:
ok